config/locales/en/providers.yml

```diff
@@ -819,12 +819,16 @@ en:
       employment_incomes:
         show:
           hmrc-information: HMRC have provided the information on this page.
-          inset_text: "We've used your client's National Insurance number to get this information from HMRC."
+          inset_text: We used your client’s National Insurance number to get this information from HMRC.
```

Comment:
The apostrophe here is different to the one on line 828

Reply:
This is already fixed in a later commit. Good spot though!